src/core/device.c

```diff
@@ -22,11 +22,10 @@
 #include <errno.h>
 #include <sys/epoll.h>
 
-#include "libudev.h"
-
 #include "alloc-util.h"
 #include "dbus-device.h"
 #include "device.h"
+#include "libudev.h"
```

Comment:
same here...

"libudev.h" is like "sd-xyz.h" one of the exported headers, and hence should really stay south of the system headers and north of our internal headers...